### DISCUSSION TITLE
2FA Account flagged/recovery

### DISCUSSION THREAD

**Discussion by v-limo** [community/community#23124] [New to GitHub]
<p>2FA Account flagged/recovery</p>
<p>My friend’s account got flagged. He contacted support and has been waiting for a while now, and he asked me to ask what he should do because he has a scheduled job interview coming soon. How long does the response normally take? And is there a way to solved this faster.</p>
<p>Ticket number 1666778</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hey there <a class="mention" href="/v-limo">@v-limo</a>!</p>
<p>Thanks for posting this on behalf of your friend. It looks like this ticket was just submitted about three days ago, but I can see that it is in the right place and assigned to a support team member. Response times may vary depending on resources and those who have tickets submitted ahead of theirs. The Community Forum is unable to escalate Support tickets so we appreciate your patience.</p>
<p>Lili</p>

**Reply by ghost** [reply]
My new account also was flagged today. Please how do I get it to be public again. It's a matter of urgency please

**Reply by tetra12** [reply]
My account has been flagged today. WTF? Are you having issues with a new security bot? Pls recover MY account to a normal operation

**Reply by BeomIn** [reply]
thank you very much!

**Comment by oolukman** [comment]
my account has been flagged too. why? i dont do anything.. what should i do?

**Comment by rabdulatipoff** [comment]
I believe that my account being flagged a few weeks ago now might be a part of this issue other users are experiencing. Could you please escalate or help support team contact yours truly through email to get this sorted? I don't recall breaking rules here... Ticket 1752308
